jmix-templates/content/flowui/list-dto/controller.java

```diff
@@ -21,7 +21,14 @@ public class ${viewControllerName} extends StandardListView<${entity.className}>
 
     @Install(to = "${tableDl}", target = Target.DATA_LOADER)
     protected List<${entity.className}> ${tableDl}LoadDelegate(LoadContext<${entity.className}> loadContext) {
-        // Here you can load entities from an external store
-        return Collections.emptyList();
+        // Here you can load entities from an external storage
```

Comment:
It is worth mentioning here that loaded DTO entities must be "not new", i.e. EntityStates.setNew(entity, false) must be invoked for them.

Reply:
done